tests/test_api.py

```diff
@@ -26,3 +24,34 @@ def test_model_parse():
         assert (rule.__class__.__name__) == "SecRule"
         for var in rule.variables:
             assert var.collection == "ARGS"
+
+
+def test_get_all_regexp() -> None:
+    """Test that we can get regular expressions correctly"""
```

Comment:
Isn't the test more about testing that `@contains` can be parsed with `>` as part of the string?

Reply:
Well, in the end, it wasn't working. We can name it just for this case, it is good.